src/plugins/cal-events/cal-events.js

```diff
@@ -372,7 +374,27 @@ var componentName = "wb-calevt",
 	};
 
 // Bind the init event of the plugin
-$document.on( "timerpoke.wb " + initEvent, selector, init );
+//$document.on( "timerpoke.wb " + initEvent, selector, init );
+
+$document.on( "timerpoke.wb " + initEvent + " wb-redraw" + selector, selector, function( event ) {
+
+	var eventType = event.type,
+		$elm = $( "#" + event.target.id ),
+		calendarId = event.currentTarget.dataset.calevtSrc;
+
+	switch ( eventType ) {
+	case "timerpoke":
+	case "wb-init":
+		init( event );
+		break;
+
+	case "wb-redraw":
+		$( "#" + calendarId + " .wb-clndr" ).remove();
```

Comment:
Can you find it relatively from the $elm?

Like
```suggestion
		$( ".wb-clndr", $elm ).remove();
```

or

```suggestion
		$elm.find( ".wb-clndr" ).remove();
```

Reply:
Unfortunately, none of those options would work as the calendar widget is added outside of the calendar of events element. So, I cannot start from the $elm reference.

Reply:
What about using the `$elm.parentUntil( ".wb-clndr" )` jquery function? That way the user/web author would not need to provide that technical information to the component which it don't provide any personalization advantage.

Reply:
well, ignore my previous comment. 
There is several example where the web author identify where to insert the calendar.